deploy/docker/Dockerfile

```diff
@@ -46,7 +45,7 @@ RUN groupadd -r -g 1000 emqx; \
 
 USER emqx
 
-VOLUME ["/opt/emqx/log", "/opt/emqx/data"]
+VOLUME ["/opt/emqx/log", "/opt/emqx/data", "/opt/emqx/etc"]
```

Comment:
The docker library team's tell me, configuration is almost never appropriate in a volume, more info: https://github.com/docker-library/official-images/pull/12212#issuecomment-1097428914